RPM/carta-backend.spec

```diff
@@ -75,41 +65,24 @@ This package provides the release version of the backend component.
 rm -rf %{NVdir}
 git clone %{url}.git %{NVdir}
 cd %{NVdir}
-git checkout v%{version}
+git checkout dev
```

Comment:
shouldn't we checkout to a tag such as 5.0.0 (or whatever tag we use for a stable release) instead of dev as it keeps evolving?